src/main/java/org/apache/commons/compress/archivers/zip/ZipFile.java

```diff
@@ -374,6 +374,8 @@ private ZipFile(final SeekableByteChannel channel, final String archiveName,
             }
             fillNameMap();
             success = true;
+        } catch (IOException e) {
```

Comment:
Does archiveName here include the path?

Reply:
If it used a constructor that gets the path sent into then yes, otherwise it will say "unknown archive"